terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -44,6 +48,15 @@ public void setDefaultTimeoutForImplicitlyWait(
     }
 
     /**
+     * Get the text (display value) set for the specified element.
+     * @param by Identifier to look for elements
+     * @return And returns the text (display value)
+     */
+    public String getText(By by) {
+    	return webDriver.findElement(by).getText();
+    }

```

Comment:
- [ ] If create `getText` method, you should correct to use this.
(for example https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/608/files#diff-202d72964264442bd02328455752ce30R52 )

I think that `terasoluna-gfw-functionaltest` is a policy to directly use the WebDriver's API except for complex logic. Please take into account the policy and correct it.